src/ManagedDripsHub.sol

```diff
@@ -128,9 +133,30 @@ abstract contract ManagedDripsHub is DripsHub, UUPSUpgradeable {
         _;
     }
 
-    /// @notice Gets the storage slot holding the paused flag.
-    function _pausedSlot() private pure returns (StorageSlot.BooleanSlot storage) {
-        return StorageSlot.getBooleanSlot(SLOT_PAUSED);
+    /// @notice Returns the DripsHub storage.
+    /// @return storageRef The storage.
+    function _dripsHubStorage()
+        internal
+        view
+        override
+        returns (DripsHubStorage storage storageRef)
+    {
+        return _managedDripsHubStorage().dripsHubStorage;
```

Comment:
I think it is a good idea to just use one storage slot for managedDripsHub and the actual dripsHub.

Since the `dripsHubStorage` is at the first position in the struct.  We could check if it gets cheaper to just return the slot in assembly here. Instead of calling the method.